src/main/java/org/fcrepo/camel/processor/ProcessorUtils.java

```diff
@@ -40,55 +42,24 @@ private ProcessorUtils() {
     }
 
     private static String trimTrailingSlash(final String path) {
-        String trimmed = path;
-        while (trimmed.endsWith("/")) {
-            trimmed = trimmed.substring(0, trimmed.length() - 1);
-        }
-        return trimmed;
-    }
-
-    /**
-     * Extract a language string suitable for Jena from a MimeType value
-     * @param contentType a MIMEType string
-     * @return a Jena-compatible language string
-     */
-    public static String langFromMimeType(final String contentType) {
-        if (contentType.equals("application/n-triples")) {
-            return "N-TRIPLE";
-        } else if (contentType.equals("text/turtle")) {
-            return "TURTLE";
-        } else if (contentType.equals("application/rdf+xml")) {
-            return "RDF/XML";
-        } else if (contentType.equals("text/rdf+n3") || contentType.equals("text/n3")) {
-            return "N3";
-        } else {
-            return null;
+        if (path.endsWith("/")) {
+            return trimTrailingSlash(path.substring(0, path.length() - 1));
```

Comment:
Eh, I have no problem with you dealing with it, but do it in a way that doesn't confuse maintainers: recurse on the input via `trimTrailingSlash`?